cmake/simd.cmake

```diff
@@ -1,12 +1,15 @@
 include(cmake/TargetArch.cmake)
 
 function(target_enable_simd tgt)
-    target_architecture(arch)
-    set(is_gcc_like "$<OR:$<COMPILE_LANG_AND_ID:CXX,AppleClang,Clang,GNU>,$<COMPILE_LANG_AND_ID:C,AppleClang,Clang,GNU>>")
-    set(is_msvc_like "$<OR:$<COMPILE_LANG_AND_ID:CXX,MSVC>,$<COMPILE_LANG_AND_ID:C,MSVC>>")
-    set(is_arch_x64 "$<STREQUAL:${arch},x86_64>")
-    target_compile_options(${tgt} PRIVATE
-            $<$<AND:${is_arch_x64},${is_gcc_like}>:-mavx2>
-            $<$<AND:${is_arch_x64},${is_msvc_like}>:/arch:AVX2>
-            )
-endfunction()
+    if(NOT APPLE)
```

Comment:
This will also prevent enabling simd on local macOS builds, but that doesn't seem like a big problem since we're likely not relying on simd too much yet and we suspect there aren't any active macOS users.

Reply:
Yeah, it's not the best situation. I suppose for this case we could add another condition [here](https://github.com/acquire-project/acquire-driver-egrabber/blob/main/cmake/TargetArch.cmake#L81) for arm64, since that CMake file appears to be pretty old. OTOH we're already disabling simd on the common and zarr drivers because the runners don't support it.